elasticsearch.yml

```diff
@@ -16,6 +17,7 @@ development:
 
 test:
   base_uri: <%= ENV.fetch('ELASTICSEARCH_URI', 'http://localhost:9200') %>
+  base_uri_b: <%= ENV["ELASTICSEARCH_B_URI"] || 'http://localhost:9200' %>
```

Comment:
Does it make sense for the A and B to actually refer to the same thing?

I feel like it'll be confusing as soon as indexing is added, and we start to add tests like "after indexing a document it can be retrieved from both clusters".

Reply:
`base_uri` is also the fallback if `base_uri_b` is unset (as is the case outside of the test environment). Rather than returning an error when say, `cluster=B` is part of the request but this isn't defined in the environment, we fall back to the default cluster.

Reply:
What do you think we should do here? This is mainly there so that any tests that want to speak to a non-default cluster can do so. When we add indexing and so on for other clusters, perhaps we should handle this better there?

Reply:
It's probably fine for now, but we'll need to rethink this somehow when indexing is added.  The easiest way to do that might be to run both ES5 and ES6 in CI (and the dev VM) on different ports.